ci/environment.yml

```
@@ -3,11 +3,11 @@ channels:
 - conda-forge
 - defaults
 dependencies:
-- pandas
-- pyomniscidb
+- python >=3.7.0
```

heavyai still needs heavydb:
```suggestion
- pyheavydb
- python >=3.7.0
```

---

It cannot be in the env yet because it's not on conda. Note: the envs are only used in CI.

---

The dependency is installed manually with pip outside of the env file. There were issues listing here with pip. But maybe was linked to something else. I can retry if you want.